documentation/style/custom.less

```diff
@@ -16,6 +16,14 @@
 @import "icons.data.svg.css";
 @import "temp.less";
 
+.attivio-primary-background {
```

Comment:
Are these styles in here twice (here and custom-general-overrides.less)?

Reply:
Yes, I forgot to remove them from the other file after adding them here.